{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:odoo:odoo:8.0:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:odoo:odoo:9.0:*:*:*:community:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:odoo:odoo:9.0:*:*:*:enterprise:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:odoo:odoo:10.0:*:*:*:community:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:odoo:odoo:10.0:*:*:*:enterprise:*:*:*",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2017-10803"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "product": {
              "product_data": [
                {
                  "product_name": "odoo",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "=",
                        "version_value": "8.0"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "9.0"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "10.0"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "odoo"
          }
        ]
      }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "In Odoo 8.0, Odoo Community Edition 9.0 and 10.0, and Odoo Enterprise Edition 9.0 and 10.0, insecure handling of anonymization data in the Database Anonymization module allows remote authenticated privileged users to execute arbitrary Python code, because unpickle is used."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-19"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://github.com/odoo/odoo/issues/17898",
          "refsource": "CONFIRM",
          "tags": [
            "Patch",
            "Third Party Advisory"
          ],
          "url": "https://github.com/odoo/odoo/issues/17898"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "accessComplexity": "MEDIUM",
        "accessVector": "NETWORK",
        "authentication": "SINGLE",
        "availabilityImpact": "COMPLETE",
        "baseScore": 8.5,
        "confidentialityImpact": "COMPLETE",
        "integrityImpact": "COMPLETE",
        "vectorString": "AV:N/AC:M/Au:S/C:C/I:C/A:C",
        "version": "2.0"
      },
      "exploitabilityScore": 6.8,
      "impactScore": 10,
      "obtainAllPrivilege": false,
      "obtainOtherPrivilege": false,
      "obtainUserPrivilege": false,
      "severity": "HIGH",
      "userInteractionRequired": true
    },
    "baseMetricV3": {
      "cvssV3": {
        "attackComplexity": "LOW",
        "attackVector": "LOCAL",
        "availabilityImpact": "HIGH",
        "baseScore": 6.5,
        "baseSeverity": "MEDIUM",
        "confidentialityImpact": "HIGH",
        "integrityImpact": "HIGH",
        "privilegesRequired": "HIGH",
        "scope": "UNCHANGED",
        "userInteraction": "REQUIRED",
        "vectorString": "CVSS:3.0/AV:L/AC:L/PR:H/UI:R/S:U/C:H/I:H/A:H",
        "version": "3.0"
      },
      "exploitabilityScore": 0.6,
      "impactScore": 5.9
    }
  },
  "lastModifiedDate": "2017-07-12T14:41Z",
  "publishedDate": "2017-07-04T18:29Z"
}